src/types/CreditCard.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ *
+ */
+
+import * as t from "io-ts";
+import { string } from "io-ts";
+
+import { NonEmptyString, PatternString } from "italia-ts-commons/lib/strings";
+
+import { PaymentMethod } from "./PaymentMethod";
+
+export type CreditCardNumber = t.TypeOf<typeof CreditCardNumber>;
+export const CreditCardNumber = PatternString("^\\d{4} \\d{4} \\d{4} \\d{4}$");
```

Comment:
why the spaces? why not `\\d{16}` for simplicity? formatting this value should be a responsibility of the UI